run.sh

```diff
@@ -20,5 +20,5 @@ echo "Installing dependencies in $DIR ..."
 export PIPENV_PIPFILE="$DIR/Pipfile"
 python3 -m pipenv install
 
-echo "Running "$1" ${@:2} ..."
+if [[ ! "$1" == *benchmark* ]]; then (echo "Running "$1" ${@:2} ..."; ); fi
```

Comment:
Why this?

Reply:
`Running "$1" ${@:2} ...` This applies to all the workflows in the build library. Avoided the logging statements only for Benchmark workflow ti hide the sensitive information.